src/pages/experiments/[experimentId]/plots-and-tables/heatmap/index.jsx

```diff
@@ -37,8 +36,6 @@ const HeatmapPlot = ({ experimentId }) => {
   const [vegaSpec, setVegaSpec] = useState();
   const displaySavedGenes = useRef(true);
 
-  const cellOptions = useSelector(getCellSetsHierarchyByType('cellSets'));
```

Comment:
why do we not need to call `getCellSetsHierarchyByType` anymore?

Reply:
This is done internally by the `SelectData` component.